app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+
+  def new
+    redirect_to home_path if current_user
+  end
+
+  def create
+    user = User.where(email: params[:email]).first
+    if user && user.authenticate(params[:password])
+      session[:user_id] = user.id
+      redirect_to home_path, notice: "You are signed in, enjoy!"
```

Comment:
You are using the `alert-notice` class here. You need to do:

```ruby
flash[:success] = "You are signed in, enjoy!"
```

instead of using notice like this. That will fix your styling issues. The same applies to all of the rest of the places you did this.